src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java

```diff
@@ -59,4 +60,17 @@ public void send(PaymentCommand cmd) {
             );
         }
     }
+
+    private void doSend(IMessage message, boolean retry) throws ServiceBusException, InterruptedException {
+        try {
+            queueClient.send(message);
+        } catch (Exception ex) {
+            if (retry) {
+                doSend(message, false);
```

Comment:
could you please add some logs so that we know the behaviour ?

Reply:
added